docs/finalproject/roles.md

```diff
@@ -11,39 +11,33 @@ These roles should in total create a [balanced team](https://www.youtube.com/wat
 ## Volunteer Roles
 
 ### Product Manager // Scrum Master
+Full role description is [here](https://docs.codeyourfuture.io/volunteers/teams-1/cyf-products-final-projects/roles/product-manager)
 
-**Time Commitment**: One day a week plus 2-4 hours during the week
-
-Scrum Masters are expected to coordinate the team and make sure everyone knows what they are working on. They can provide support in
-
+Your main role is to coordinate the team and make sure everyone knows what they are working on. They can provide support in:
 - Scheduling meetings
-- Working with the team to create and update tasks on Trello
+- Working with the team to create and update tasks on Github Projects
 - Introducing and enforcing Agile practices (e.g. sprint planning, retros)
 
-### Technical Assistant
-
-**Time Commitment**: One day a week plus 2-4 hours during the week
-
-Teaching assistant provide technical support to the trainees as they are developing their projects. They can provide support in
+### Technical Lead
+Full role description and commitment is [here](https://docs.codeyourfuture.io/volunteers/teams-1/cyf-products-final-projects/roles/tech-lead)
```

Comment:
```suggestion
Full [role description for Tech Lead](https://docs.codeyourfuture.io/volunteers/teams-1/cyf-products-final-projects/roles/tech-lead) (TL)
```
- Links must make sense out of context.  https://quail.readthedocs.io/en/latest/tests/aLinksMakeSenseOutOfContext.html
- Acronyms must be introduced once before they can be used
https://www.w3.org/TR/2016/NOTE-UNDERSTANDING-WCAG20-20161007/meaning-located.html